test/corpus/glimmer.txt

```diff
@@ -0,0 +1,136 @@
+============================================
+Simple
+============================================
+
+const Named = <template>
+  {{ (doubled foo) }}
+</template>
+
+----
+
+(program
+  (lexical_declaration
+    (variable_declarator
+      (identifier)
+      (glimmer_template
+        (glimmer_opening_tag)
+        (glimmer_closing_tag)))))
+
+
+============================================
+Empty
+============================================
+
+<template></template>
+
+----
+
+(program
+  (expression_statement
+    (glimmer_template
+      (glimmer_opening_tag)
+      (glimmer_closing_tag))))
+
+
+============================================
+Two Components
+============================================
+
+const WithSemi = <template>
+  {{ (doubled foo) }}
+</template>;
+
+<template>
+  <WithSemi />
```

Comment:
So whatever is inside the <template> is not really parsed/recognized?

Reply:
Correct, that's handled by tree-sitter-glimmer via injections